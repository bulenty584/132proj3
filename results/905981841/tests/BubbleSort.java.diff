1,17c1,22
< Exception in thread "main" ParseException: Encountered " <INTEGER_LITERAL> "1 "" at line 114, column 26.
< Was expecting one of:
<     <IDENTIFIER> ...
<     <IDENTIFIER> ...
<     <IDENTIFIER> ...
<     <IDENTIFIER> ...
<     <IDENTIFIER> ...
<     
< 	at SparrowParser.generateParseException(SparrowParser.java:1202)
< 	at SparrowParser.jj_consume_token(SparrowParser.java:1084)
< 	at SparrowParser.Identifier(SparrowParser.java:506)
< 	at SparrowParser.Add(SparrowParser.java:224)
< 	at SparrowParser.Instruction(SparrowParser.java:123)
< 	at SparrowParser.Block(SparrowParser.java:84)
< 	at SparrowParser.FunctionDeclaration(SparrowParser.java:66)
< 	at SparrowParser.Program(SparrowParser.java:23)
< 	at Main.main(Main.java:24)
---
> 20
> 7
> 12
> 18
> 2
> 11
> 6
> 9
> 19
> 5
> 99999
> 2
> 5
> 6
> 7
> 9
> 11
> 12
> 18
> 19
> 20
> 0
