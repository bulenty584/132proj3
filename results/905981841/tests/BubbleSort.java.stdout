func Main()
  y0 = 0
  v0 = 12
  v1 = alloc(v0)
  v2 = 16
  vmt_BBS = alloc(v2)
  v3 = @BBS_Print
  [vmt_BBS + 0] = v3
  v4 = @BBS_Init
  [vmt_BBS + 4] = v4
  v5 = @BBS_Start
  [vmt_BBS + 8] = v5
  v6 = @BBS_Sort
  [vmt_BBS + 12] = v6
  [v1 + 0] = vmt_BBS
  if0 v1 goto v1Error_0
  goto v1End_1
v1Error_0:
  error("null pointer")
v1End_1:
  if0 v1 goto v1Error_2
  goto v1End_3
v1Error_2:
  error("null pointer")
v1End_3:
  v7 = 10
  v8 = [v1 + 0]
  v9 = [v8 + 8]
  v10 = call v9(v1 v7)
  print(v10)
  return y0
func BBS_Start(this sz)
  if0 this goto thisError_4
  goto thisEnd_5
thisError_4:
  error("null pointer")
thisEnd_5:
  v11 = [this + 0]
  v12 = [v11 + 4]
  v13 = call v12(this sz)
  aux01 = v13
  if0 this goto thisError_6
  goto thisEnd_7
thisError_6:
  error("null pointer")
thisEnd_7:
  v14 = [this + 0]
  v15 = [v14 + 0]
  v16 = call v15(this)
  aux01 = v16
  v17 = 99999
  print(v17)
  if0 this goto thisError_8
  goto thisEnd_9
thisError_8:
  error("null pointer")
thisEnd_9:
  v18 = [this + 0]
  v19 = [v18 + 12]
  v20 = call v19(this)
  aux01 = v20
  if0 this goto thisError_10
  goto thisEnd_11
thisError_10:
  error("null pointer")
thisEnd_11:
  v21 = [this + 0]
  v22 = [v21 + 0]
  v23 = call v22(this)
  aux01 = v23
  v24 = 0
  return v24
func BBS_Sort(this)
  v25 = 1
  v26 = size - v25
  i = v26
  v27 = 0
  v28 = 1
  v29 = v27 - v28
  aux02 = v29
  v30 = aux02 < i
  v31 = 1
  j = v31
  v32 = 1
  v33 = i + v32
  v34 = j < v33
  v35 = 1
  v36 = j - v35
  aux07 = v36
  v37 = [number + 0]
  v39 = 1
  v40 = 0
  v38 = v40 - v39
  v41 = 4
  v42 = v38 < aux07
  v43 = aux07 < v37
  v44 = v42 * v43
  if0 v44 goto v44Error_12
  goto v44End_13
v44Error_12:
  error("array index out of bounds")
v44End_13:
  v45 = aux07 * v41
  v46 = v45 + v41
  v47 = number + v46
  v48 = [v47 + 0]
  aux04 = v48
  v49 = [number + 0]
  v51 = 1
  v52 = 0
  v50 = v52 - v51
  v53 = 4
  v54 = v50 < j
  v55 = j < v49
  v56 = v54 * v55
  if0 v56 goto v56Error_14
  goto v56End_15
v56Error_14:
  error("array index out of bounds")
v56End_15:
  v57 = j * v53
  v58 = v57 + v53
  v59 = number + v58
  v60 = [v59 + 0]
  aux05 = v60
  if0 null goto else_17
  v61 = aux05 < aux04
  goto ifend_18
  else_17:
  v62 = 1
  v63 = j - v62
  aux06 = v63
  v64 = [number + 0]
  v66 = 1
  v67 = 0
  v65 = v67 - v66
  v68 = 4
  v69 = v65 < aux06
  v70 = aux06 < v64
  v71 = v69 * v70
  if0 v71 goto v71Error_19
  goto v71End_20
v71Error_19:
  error("array index out of bounds")
v71End_20:
  v72 = aux06 * v68
  v73 = v72 + v68
  v74 = number + v73
  v75 = [v74 + 0]
  t = v75
  v76 = [number + 0]
  v78 = 1
  v79 = 0
  v77 = v79 - v78
  v80 = 4
  v81 = v77 < j
  v82 = j < v76
  v83 = v81 * v82
  if0 v83 goto v83Error_21
  goto v83End_22
v83Error_21:
  error("array index out of bounds")
v83End_22:
  v84 = j * v80
  v85 = v84 + v80
  v86 = number + v85
  v87 = [v86 + 0]
  v88 = [number + 0]
  v89 = 0
  v90 = v89 < aux06
  v91 = aux06 < v88
  v92 = v90 * v91
  if0 v92 goto v92Error_23
  goto v92End_24
v92Error_23:
  error("array index out of bounds")
v92End_24:
  v94 = 1
  v93 = aux06v94
  v96 = 4
  v95 = v93 * v96
  v97 = number + v95
  [v97 + 0] = v87
  v98 = [number + 0]
  v99 = 0
  v100 = v99 < j
  v101 = j < v98
  v102 = v100 * v101
  if0 v102 goto v102Error_25
  goto v102End_26
v102Error_25:
  error("array index out of bounds")
v102End_26:
  v104 = 1
  v103 = jv104
  v106 = 4
  v105 = v103 * v106
  v107 = number + v105
  [v107 + 0] = t
  ifend_18:
  v108 = 1
  v109 = j + v108
  j = v109
  v110 = 1
  v111 = i - v110
  i = v111
  v112 = 0
  return v112
func BBS_Print(this)
  v113 = 0
  j = v113
  v114 = j < size
  v115 = [number + 0]
  v117 = 1
  v118 = 0
  v116 = v118 - v117
  v119 = 4
  v120 = v116 < j
  v121 = j < v115
  v122 = v120 * v121
  if0 v122 goto v122Error_27
  goto v122End_28
v122Error_27:
  error("array index out of bounds")
v122End_28:
  v123 = j * v119
  v124 = v123 + v119
  v125 = number + v124
  v126 = [v125 + 0]
  print(v126)
  v127 = 1
  v128 = j + v127
  j = v128
  v129 = 0
  return v129
func BBS_Init(this sz)
  size = sz
  v131 = 1
  v130 = sz + v131
  v133 = 4
  v132 = v130 * v133
  v134 = alloc(v132)
  if0 v134 goto v134Error_29
  goto v134End_30
v134Error_29:
  error("null pointer")
v134End_30:
  [v134 + 0] = sz
  number = v134
  v135 = 0
  v136 = 20
  v137 = [number + 0]
  v138 = 0
  v139 = v138 < v135
  v140 = v135 < v137
  v141 = v139 * v140
  if0 v141 goto v141Error_31
  goto v141End_32
v141Error_31:
  error("array index out of bounds")
v141End_32:
  v143 = 1
  v142 = v135v143
  v145 = 4
  v144 = v142 * v145
  v146 = number + v144
  [v146 + 0] = v136
  v147 = 1
  v148 = 7
  v149 = [number + 0]
  v150 = 0
  v151 = v150 < v147
  v152 = v147 < v149
  v153 = v151 * v152
  if0 v153 goto v153Error_33
  goto v153End_34
v153Error_33:
  error("array index out of bounds")
v153End_34:
  v155 = 1
  v154 = v147v155
  v157 = 4
  v156 = v154 * v157
  v158 = number + v156
  [v158 + 0] = v148
  v159 = 2
  v160 = 12
  v161 = [number + 0]
  v162 = 0
  v163 = v162 < v159
  v164 = v159 < v161
  v165 = v163 * v164
  if0 v165 goto v165Error_35
  goto v165End_36
v165Error_35:
  error("array index out of bounds")
v165End_36:
  v167 = 1
  v166 = v159v167
  v169 = 4
  v168 = v166 * v169
  v170 = number + v168
  [v170 + 0] = v160
  v171 = 3
  v172 = 18
  v173 = [number + 0]
  v174 = 0
  v175 = v174 < v171
  v176 = v171 < v173
  v177 = v175 * v176
  if0 v177 goto v177Error_37
  goto v177End_38
v177Error_37:
  error("array index out of bounds")
v177End_38:
  v179 = 1
  v178 = v171v179
  v181 = 4
  v180 = v178 * v181
  v182 = number + v180
  [v182 + 0] = v172
  v183 = 4
  v184 = 2
  v185 = [number + 0]
  v186 = 0
  v187 = v186 < v183
  v188 = v183 < v185
  v189 = v187 * v188
  if0 v189 goto v189Error_39
  goto v189End_40
v189Error_39:
  error("array index out of bounds")
v189End_40:
  v191 = 1
  v190 = v183v191
  v193 = 4
  v192 = v190 * v193
  v194 = number + v192
  [v194 + 0] = v184
  v195 = 5
  v196 = 11
  v197 = [number + 0]
  v198 = 0
  v199 = v198 < v195
  v200 = v195 < v197
  v201 = v199 * v200
  if0 v201 goto v201Error_41
  goto v201End_42
v201Error_41:
  error("array index out of bounds")
v201End_42:
  v203 = 1
  v202 = v195v203
  v205 = 4
  v204 = v202 * v205
  v206 = number + v204
  [v206 + 0] = v196
  v207 = 6
  v208 = 6
  v209 = [number + 0]
  v210 = 0
  v211 = v210 < v207
  v212 = v207 < v209
  v213 = v211 * v212
  if0 v213 goto v213Error_43
  goto v213End_44
v213Error_43:
  error("array index out of bounds")
v213End_44:
  v215 = 1
  v214 = v207v215
  v217 = 4
  v216 = v214 * v217
  v218 = number + v216
  [v218 + 0] = v208
  v219 = 7
  v220 = 9
  v221 = [number + 0]
  v222 = 0
  v223 = v222 < v219
  v224 = v219 < v221
  v225 = v223 * v224
  if0 v225 goto v225Error_45
  goto v225End_46
v225Error_45:
  error("array index out of bounds")
v225End_46:
  v227 = 1
  v226 = v219v227
  v229 = 4
  v228 = v226 * v229
  v230 = number + v228
  [v230 + 0] = v220
  v231 = 8
  v232 = 19
  v233 = [number + 0]
  v234 = 0
  v235 = v234 < v231
  v236 = v231 < v233
  v237 = v235 * v236
  if0 v237 goto v237Error_47
  goto v237End_48
v237Error_47:
  error("array index out of bounds")
v237End_48:
  v239 = 1
  v238 = v231v239
  v241 = 4
  v240 = v238 * v241
  v242 = number + v240
  [v242 + 0] = v232
  v243 = 9
  v244 = 5
  v245 = [number + 0]
  v246 = 0
  v247 = v246 < v243
  v248 = v243 < v245
  v249 = v247 * v248
  if0 v249 goto v249Error_49
  goto v249End_50
v249Error_49:
  error("array index out of bounds")
v249End_50:
  v251 = 1
  v250 = v243v251
  v253 = 4
  v252 = v250 * v253
  v254 = number + v252
  [v254 + 0] = v244
  v255 = 0
  return v255
